packages/preview/sweet-graduate-resume/0.1.0/template.typ

```diff
@@ -0,0 +1,139 @@
+#import "lib_fontawesome.typ": *
+
+#let preamble(doc) = {
+  set page(
+    paper: "a4",
+    margin: (x: 0.5cm, y: 0.5cm),
+  )
+
+  set text(
+    font: "DejaVu Sans",
+    size: 8pt,
+  )
+
+  set par(justify: true)
+
+  show link: name => {
+    set text(blue)
+    name
+  }
+
+  set block(spacing: 0.5em)
+
+  set align(left)
+  doc
+}
+
+#let header(author, roll, school, urls) = {
+  set text(17pt, weight: "bold")
+  set block(spacing: 0.3em)
+  [
+    #author | #roll
+
+    #set text(13pt)
+    #box(
+      width: 1fr,
+      inset: (bottom: 5pt),
+      stroke: (bottom: (1pt + black)),
+    )[#school]
+
+  ]
+
+  set text(7pt)
+  for value in urls {
+    let img = if value.fa {
+      if value.brand {
+        fa-icon(value.svg)
+      } else {
+        fa-icon(value.svg, solid: value.solid)
+      }
+    } else {
+      image("svg/" + value.svg)
+    }
+
+    link(value.url)[
+      #value.name
+      #box(height: 1em, baseline: 20%)[#img]
+    ] + if value == urls.last() {
+      ""
+    } else {
+      "|"
+    }
+  }
+}
+
+#let section_header(head) = {
```

Comment:
Typst packages generally follow the kebab-case naming convention. I would recommend to rename public function and their arguments to use that instead of snake_case. However, the choice is up to you, and if you really prefer snake_case, just tell me, it won't block the package from being merged.

Reply:
It was a force of habit due to rust conventions. I believe the latest commit should fix it.